src/dra/4A538.c

```diff
@@ -1131,7 +1131,7 @@ void FreePrimitives(s32 primitiveIndex) {
     if (prim) {
         do {
             if (prim->type == PRIM_ENV) {
-                *(*(s32**)&prim->r1) = 0;
+                **(DR_ENV***)&prim->r1 = NULL;
```

Comment:
Surprised by the *** but if it's running I guess it's correct

Reply:
Unfortunately I couldn't match it otherwise. Yes, it is very odd.